services/admin/src/admin/routes/backfill.py

```diff
@@ -54,7 +54,7 @@ async def backfill_endpoint(request: Request) -> Response:
             )
         except AdminCustomError as e:
             return get_json_admin_error_response(e, max_age=0)
-        except Exception:
-            return get_json_admin_error_response(UnexpectedError("Unexpected error."), max_age=0)
+        except Exception as e:
+            return get_json_admin_error_response(UnexpectedError("Unexpected error.", e), max_age=0)
```

Comment:
We used to log the error type and error message as well in `datasets`. What do you think?
```suggestion
            return get_json_admin_error_response(UnexpectedError(f"Unexpected error {type(e)}: {e}", e), max_age=0)
```

If you agree, then you should update the other lines accordingly.

Reply:
Hmmm, this message will be shown to the users in the dataset viewer, and in the API responses. We don't want to show internal errors, only user-related errors. At least, it's how I've created the error system for now. That's why I don't put details in the message, but the traceback in the logs.

Reply:
I see your point, yes I agree.